locales/en/questsContent.json

```diff
@@ -72,7 +72,7 @@
 
   "questVice3Text": "Vice Awakens",
   "questVice3Notes": "After much effort, your party has discovered Vice's lair. The hulking monster eyes your party with distaste. As shadows swirl around you, a voice whispers through your head, \"More foolish citizens of Habitica come to stop me? Cute. You'd have been wise not to come.\" The scaly titan rears back its head and prepares to attack. This is your chance! Give it everything you've got and defeat Vice once and for all!",
-  "questVice3Completion": "The shadows dissipate from the cavern and a steely silence falls. My word, you've done it! You have defeated Vice! You and your party may finally breath a sigh of relief. Enjoy your victory, brave Habiteers, but take the lessons you've learned from battling Vice and move forward. There are still habits to be done and potentially worse evils to conquer!",
+  "questVice3Completion": "The shadows dissipate from the cavern and a steely silence falls. My word, you've done it! You have defeated Vice! You and your party may finally breath a sigh of relief. Enjoy your victory, brave Habiteers, but take the lessons you've learned from battling Vice and move forward. There are still Habits to be done and potentially worse evils to conquer!",
```

Comment:
Ooh, this one is tricky. "Habits to be done" could go either way. Since real life habits are those done continually, to make them done seems more of like an end goal. IE, in HabitRPG, you make Habits with the hope that some day, you'll be done with them and can remove them from the app.

I don't know, someone else want to weigh in?